lib/safemode/parser.rb

```diff
@@ -159,25 +162,39 @@ def process_call_args(exp)
         end
         args << processed unless (processed.nil? or processed.empty?)
       end
-      args.empty? ? nil : args.join(", ")
+      args
     end
 
-    def process_call_code(receiver, name, args)
+    def process_call_code(receiver, name, args, safe_call)
       case name
-      when :<=>, :==, "!=".to_sym, :<, :>, :<=, :>=, :-, :+, :*, :/, :%, :<<, :>>, :** then
```

Comment:
Looking at https://github.com/seattlerb/ruby2ruby/blob/ddf539eaf348b57c35acd91109fc6d368289b517/lib/ruby2ruby.rb#L39-L40 this implies we'll also support `|` and `&` operators which previously weren't supported. I'm not sure if there's a security risk involved there.

Reply:
For the sake of completeness, we will support `^` as well. How would someone misuse these operators?.. @adamruzicka, maybe you have an idea? :)

Reply:
Looking at https://github.com/theforeman/safemode/blob/01af658932811800e2fc0542db4332a922192131/lib/safemode/core_jails.rb#L30 I'd say we did support them even before, but in a non-operator kind of way?

Before this change when I try to render `<%= 5 ^ 3 %>`
```
> receiver
=> "5.to_jail"
> name
=> :^
> args
=> "3"
> "#{receiver}.#{name}#{args ? "(#{args})" : args}"
=> "5.to_jail.^(3)"
```

With this change the generated code will be `5.to_jail ^ 3`, but semantically it should be the same thing so we don't really need to worry about it?